engine/src/cmd/container.h

```diff
@@ -8,7 +8,7 @@ class Unit;
 class UnitContainer
 {
 protected:
-    Unit *unit;
+    Unit *unit = nullptr;
```

Comment:
this should be done in the constructor/implementation file, not the header file

Reply:
Again, I think I disagree. From what I've been reading, in C++11 and later, initializing fields in the header file is acceptable, even preferable.

Reply:
Missed that one.